composer.json

```diff
@@ -20,19 +20,21 @@
         }
     ],
     "require": {
-        "php": "^8.0",
-        "illuminate/database": "^9.0|^10.0",
-        "illuminate/http": "^9.0|^10.0",
-        "illuminate/support": "^9.0|^10.0",
+        "php": "^8.2",
```

Comment:
Deze is nu wel maar php 8.2 compatible moet dit geen 8.3 zijn?

Reply:
Zoals besproken hoeft dat niet vanwege ^

Reply:
Zoals besproken hoeft dat niet vanwege ^